src/style.css

```diff
@@ -1 +1,76 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+li {
+  list-style: none;
+}
+
+li:not(:first-child) {
+  margin-left: 20px;
+}
+
+.myHeader {
```

Comment:
Please, check all of you class names - typical pattern "some-name" and it should consist only of semantic meanings - "my" is redundant here. You can take names examples from here https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md - second item